src/reducers/Setsigninerrorreducer.js

```diff
@@ -0,0 +1,11 @@
+import { SET_ERROR_SIGNIN } from '../actions/index';
```

Comment:
It's better if we have a single "SET_ERROR" action for the whole app instead of having separate SET_ERROR_SIGNIN and SET_ERROR_LOGIN (and what's the difference between login and signin? They seem to mean the same thing)